### ISSUE TITLE
Punycode domains not working

### ISSUE THREAD

**Issue by devvick** [Athou/commafeed#1588]
**Describe the bug**
I cannot add a URL with punycode domain as feed.

**To Reproduce**

1. Try adding https://xn--gckvb8fzb.com/index.xml as feed.
2. `Error: jakarta.ws.rs.WebApplicationException: マリウス.com: Name or service not known`

**Expected behavior**
Able to add a punycode domain as feed

**Environment (please complete the following information):**
- self-hosted (official Docker image **-jvm** version)
CommaFeed version 5.3.3 (a337b01).

**Additional info**
The domain is resolvable from the same host:
```bash
$ drill xn--gckvb8fzb.com
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 6792
;; flags: qr rd ra ; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; xn--gckvb8fzb.com.	IN	A

;; ANSWER SECTION:
xn--gckvb8fzb.com.	300	IN	A	143.244.50.83
```

**Comment by Athou** [comment]
I released 5.3.4 with a fix for this, thank you for the bug report!

**Comment by devvick** [comment]
Thank you, that was incredibly fast  :dash:
